docs/api.md

```diff
@@ -1310,10 +1310,12 @@ Generate a bash completion script. Users of your application can install this
 script in their `.bashrc`, and yargs will provide completion shortcuts for
 commands and options.
 
-.showHelp(consoleLevel='error')
+.showHelp()
```

Comment:
To be consistent with the rest of the document I might put something like:

`.showHelp([level | printCallback])`

If no argument is provided ... your description.

If a string argument is provided ... your description

If a `printCallback` is provided ... your description.


thanks for your help fleshing out these docs :ok_hand:

Reply:
Fixed according to your suggestion.

Not sure the docs are very consistent in this matter.

Examples of the "multiple line" syntax: [.help](https://github.com/yargs/yargs/blob/master/docs/api.md#help), [.showHidden](https://github.com/yargs/yargs/blob/master/docs/api.md#showhidden), [.version](https://github.com/yargs/yargs/blob/master/docs/api.md#version).

I could only find a few examples of the "pipe" syntax you suggested, and they were used in scenarios with a matrix of options: `.help`, `.showHidden`, [.require/.required](https://github.com/yargs/yargs/blob/master/docs/api.md#requirekey-msg--boolean), [.usage](https://github.com/yargs/yargs/blob/master/docs/api.md#usagemessagecommand-desc-builder-handler), `.version`.

Perhaps we need a stricter syntax? We can use the [typings from @types/yargs package](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/yargs/index.d.ts) as a starting point.

Reply:
I agree with @royra about the need of not reinventing a syntax of our own, and using widely used syntax as typescript or [jsdoc](https://jsdoc.app/tags-type.html).

I would suggest merging this PR as is, and opening a separate issue to review all docs to apply a standard typing syntax.

@bcoe are you OK with this?